src/bp-core/admin/js/settings-page.js

```diff
@@ -692,6 +699,7 @@
 									'action': 'bp_admin_forum_repair_tools_wrapper_function',
 									'type': BbToolsForumsRepairActions[currentAction],
 									'offset': offset,
+									'site_id': SiteId,
```

Comment:
@janakkaneriya007 `site_id` is never used in the backend in this PR. is it working?

Reply:
why do we need global variable for site id?  just get the value of site from the dropdown `$('#select-site').val()` when clicked button or doing ajax request.